client/me/billing-history/style.scss

```diff
@@ -192,6 +192,18 @@
 	min-height: 1.2em;
 }
 
+.billing-history__billing-details-editable {
+	&:hover {
+		outline: 1px black dashed;
```

Comment:
I changed this to an outline so that the text doesn't move around when the box is focussed.